app/main.py

```diff
@@ -1,6 +1,24 @@
+import requests
+import os
+
+
+API_KEY = os.environ["weatherapi_key"]
+URL = f"https://api.weatherapi.com/v1/current.json?key={API_KEY}"
+
+FILTERING = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    data = requests.get(f"{URL}&q={FILTERING}")
```

Comment:
Please check previous comments and fix them. Use `params` for passing query parameters and don't touch requirements.txt